app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/partnership/PartnershipPartnersListController.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.controllers.partnership
+
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.plasticpackagingtax.registration.connectors._
+import uk.gov.hmrc.plasticpackagingtax.registration.controllers.actions.AuthAction
+import uk.gov.hmrc.plasticpackagingtax.registration.models.registration.Cacheable
+import uk.gov.hmrc.plasticpackagingtax.registration.models.request.JourneyAction
+import uk.gov.hmrc.plasticpackagingtax.registration.views.html.partnerships.other_partners_page
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.ExecutionContext
+
+@Singleton
+class PartnershipPartnersListController @Inject() (
+  authenticate: AuthAction,
+  journeyAction: JourneyAction,
+  override val registrationConnector: RegistrationConnector,
+  mcc: MessagesControllerComponents,
+  otherPartnersPage: other_partners_page
+)(implicit val executionContext: ExecutionContext)
+    extends FrontendController(mcc) with Cacheable with I18nSupport {
+
+  def displayPage(): Action[AnyContent] =
+    (authenticate andThen journeyAction) { implicit request =>
+      if (request.registration.isPartnership)
+        (for {
+          partnershipDetails <- request.registration.organisationDetails.partnershipDetails
+          nominatedPartnerName = request.registration.primaryContactDetails.name.getOrElse(
+            "Nominated partner name"
+          ) // TODO Enforce when available
```

Comment:
We expect this partner name to be available -somewhere- on registration after the GRS stop.